src/plugins/graph/models/graph-model.ts

```diff
@@ -470,9 +472,27 @@ export const GraphModel = TileContentModel
       const colorIndex = self._idColors.get(id);
       if (colorIndex === undefined) return "black";
       return clueGraphColors[colorIndex % clueGraphColors.length].name;
+    },
+    get isShowingMovableLine() {
+      return self.adornments.find(a => a.type === kMovableLineType)?.isVisible;
     }
   }))
   .actions(self => ({
+    showMovableLine() {
+      const mLine = MovableLineModel.create();
+      mLine.setInitialLine(self.axes.get("bottom"), self.axes.get("left"), "{}");
```

Comment:
Note that this model will be created on every invocation of this method, even though it will only be used on the first invocation.